modules/Sources/Features/SecurityWarning/SecurityWarningView.swift

```diff
@@ -32,13 +32,18 @@ public struct SecurityWarningView: View {
                 Text(L10n.SecurityWarning.title)
                     .font(.custom(FontFamily.Archivo.semiBold.name, size: 25))
                     .padding(.bottom, 8)
-                
-                Text(L10n.SecurityWarning.warningPart1(viewStore.appVersion, viewStore.appBuild))
-                    .font(.custom(FontFamily.Inter.medium.name, size: 16))
-                + Text(L10n.SecurityWarning.warningPart2)
-                    .font(.custom(FontFamily.Inter.bold.name, size: 16))
-                + Text(L10n.SecurityWarning.warningPart3)
-                    .font(.custom(FontFamily.Inter.medium.name, size: 16))
+
+                Group {
+                    Text(L10n.SecurityWarning.warningPart1a(viewStore.appVersion, viewStore.appBuild))
+                    + Text("[\(L10n.SecurityWarning.warningPart1b)](https://z.cash/privacy-policy/)")
+                        .underline()
+                    + Text(L10n.SecurityWarning.warningPart1c)
+                    + Text(L10n.SecurityWarning.warningPart2)
+                        .font(.custom(FontFamily.Inter.bold.name, size: 16))
+                    + Text(L10n.SecurityWarning.warningPart3)
+                }
```

Comment:
`Group` doesn't have any affect on a Text inside or any view inside. The whole purpose of the group here is to satisfy the compiler because SwiftUI requires an opaque type to be returned. Once there is more than 1 type of views, the developer needs to solve this problem by wrapping several views into a group. Layout or rendering of the content inside the group is not influenced here.